sklearn/preprocessing/data.py

```diff
@@ -1698,14 +1698,11 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
-    n_values : 'auto', int or array of ints
-        Number of values per feature.
+    classes : 'auto', 2D array of ints or strings or both.
```

Comment:
@jnothman So if a feature can take negative values the programmer should pass a list of lists ?